packages/terra-clinical-item-collection/src/Item.jsx

```diff
@@ -0,0 +1,160 @@
+   * isTruncated and accessoryAlignment.
+   */
+  listItemStyles: PropTypes.shape({
+    layout: PropTypes.oneOf(['oneColumn', 'twoColumns']),
+    textEmphasis: PropTypes.oneOf(['default', 'start']),
+    isTruncated: PropTypes.bool,
+    accessoryAlignment: PropTypes.oneOf(['alignTop', 'alignCenter']),
+  }),
+  /**
+   * Wether or not the item is selectable. If true, the item is given list and table hover and focus styles, tabIndex
+   * set to 0, and onClick and onKeyDown callbacks set to the onSelect function provided via Item Collection.
+   */
+  isSelectable: PropTypes.bool,
+  /*
+   * Wether or not the item is selected.
+   */
+  isSelected: PropTypes.bool,
+  /**
+   * The view in which the item should be presented. Options are list or table and this will be set by the Item Collection component.
+   */
+  view: PropTypes.oneOf(['list', 'table']),
+};
+
+const defaultProps = {
+  startAccessory: undefined,
+  comment: undefined,
+  endAccessory: undefined,
+  listItemStyles: {},
+  isSelectable: false,
+  isSelected: false,
+  view: 'list',
+};
+
+function createSelectableProps(onSelect, itemKey) {
+  const selectableProps = { isSelectable: true };
+  selectableProps.tabIndex = 0;
+
+  if (!itemKey) {
+    /* eslint-disable no-console */
+    console.error('Key is required for correct selectable implementation.');
+    /* eslint-disable no-console */
+  }
+
+  if (onSelect) {
+    selectableProps.onClick = (event) => {
+      onSelect(event, itemKey);
+    };
+
+    selectableProps.onKeyDown = (event) => {
+      onSelect(event, itemKey);
+    };
+  }
+
+  return selectableProps;
+}
+
+function createListItem(elements, itemKey, selectableProps, isSelected, listItemStyles) {
+  const { startAccessory, children, comment, endAccessory, hasStartAccessory, hasEndAccessory, ...listItemProps } = elements;
+  // hasStartAccessorySpacing={hasStartAccessory}
+  // hasEndAccessorySpacing={hasEndAccessory}
+
+  const listItemContent = (
+    <ItemView
+      startAccessory={startAccessory && <div>{startAccessory}</div>}
+      displays={React.Children.toArray(children)}
+      comment={comment && <div>{comment}</div>}
+      endAccessory={endAccessory && <div>{endAccessory}</div>}
+      {...listItemStyles}
+    />
+  );
+
+  return (
+    <List.Item
+      key={itemKey}
+      content={listItemContent}
+      isSelected={isSelected}
+      {...selectableProps}
+      {...listItemProps}
+    />
+  );
+}
+
+function createTableCell(content, keyValue, contentType) {
+  return (<Table.Cell content={content} key={keyValue} className={cx(`content-${contentType}`)} />);
+}
+
+function createTableRow(elements, itemKey, selectableProps, isSelected) {
+  const { startAccessory, children, comment, endAccessory, ...tableRowProps } = elements;
+
+// QUESTION:
+  // Does ITEM as a standalone component want to care about the maximun number of displays columns?
```

Comment:
I would expect ITEM to be agnostic and not care what it is being placed inside and let the container handle that logic